hawc/apps/study/forms.py

```diff
@@ -146,6 +148,63 @@ def setHelper(self):
         return super().setHelper(inputs)
 
 
+class IdentifierStudyForm(forms.Form):
+    db_type = forms.ChoiceField(
+        label="Database", required=True, choices=ReferenceDatabase.import_choices(),
+    )
+    db_id = forms.CharField(label="Database ID", required=True)
+
+    def __init__(self, *args, **kwargs):
+        kwargs.pop("instance")
```

Comment:
I had to pop `instance` since this is a form, not a model form, to play nicely with django's base create view